src/locales/cy/translation.json

```diff
@@ -245,6 +245,32 @@
         }
       }
     },
+    "noPhotoIdExitFindAnotherWay": {
```

Comment:
I have duplicated because I intend to remove the previous content references e.g. pyiEscapeM2b